assets/block_entities.json

```diff
@@ -1,227 +1 @@
-  },
-  {
-    "id": 25,
-    "ident": "minecraft:bed",
-    "name": "bed"
-  },
-  {
-    "id": 26,
-    "ident": "minecraft:conduit",
-    "name": "conduit"
-  },
-  {
-    "id": 27,
-    "ident": "minecraft:barrel",
-    "name": "barrel"
-  },
-  {
-    "id": 28,
-    "ident": "minecraft:smoker",
-    "name": "smoker"
-  },
-  {
-    "id": 29,
-    "ident": "minecraft:blast_furnace",
-    "name": "blast_furnace"
-  },
-  {
-    "id": 30,
-    "ident": "minecraft:lectern",
-    "name": "lectern"
-  },
-  {
-    "id": 31,
-    "ident": "minecraft:bell",
-    "name": "bell"
-  },
-  {
-    "id": 32,
-    "ident": "minecraft:jigsaw",
-    "name": "jigsaw"
-  },
-  {
-    "id": 33,
-    "ident": "minecraft:campfire",
-    "name": "campfire"
-  },
-  {
-    "id": 34,
-    "ident": "minecraft:beehive",
-    "name": "beehive"
-  },
-  {
-    "id": 35,
-    "ident": "minecraft:sculk_sensor",
-    "name": "sculk_sensor"
-  },
-  {
-    "id": 36,
-    "ident": "minecraft:calibrated_sculk_sensor",
-    "name": "calibrated_sculk_sensor"
-  },
-  {
-    "id": 37,
-    "ident": "minecraft:sculk_catalyst",
-    "name": "sculk_catalyst"
-  },
-  {
-    "id": 38,
-    "ident": "minecraft:sculk_shrieker",
-    "name": "sculk_shrieker"
-  },
-  {
-    "id": 39,
-    "ident": "minecraft:chiseled_bookshelf",
-    "name": "chiseled_bookshelf"
-  },
-  {
-    "id": 40,
-    "ident": "minecraft:brushable_block",
-    "name": "brushable_block"
-  },
-  {
-    "id": 41,
-    "ident": "minecraft:decorated_pot",
-    "name": "decorated_pot"
-  },
-  {
-    "id": 42,
-    "ident": "minecraft:crafter",
-    "name": "crafter"
-  },
-  {
-    "id": 43,
-    "ident": "minecraft:trial_spawner",
-    "name": "trial_spawner"
-  },
-  {
-    "id": 44,
-    "ident": "minecraft:vault",
-    "name": "vault"
-  }
-]
\ No newline at end of file
+{"minecraft:furnace":0,"minecraft:chest":1,"minecraft:trapped_chest":2,"minecraft:ender_chest":3,"minecraft:jukebox":4,"minecraft:dispenser":5,"minecraft:dropper":6,"minecraft:sign":7,"minecraft:hanging_sign":8,"minecraft:mob_spawner":9,"minecraft:creaking_heart":10,"minecraft:piston":11,"minecraft:brewing_stand":12,"minecraft:enchanting_table":13,"minecraft:end_portal":14,"minecraft:beacon":15,"minecraft:skull":16,"minecraft:daylight_detector":17,"minecraft:hopper":18,"minecraft:comparator":19,"minecraft:banner":20,"minecraft:structure_block":21,"minecraft:end_gateway":22,"minecraft:command_block":23,"minecraft:shulker_box":24,"minecraft:bed":25,"minecraft:conduit":26,"minecraft:barrel":27,"minecraft:smoker":28,"minecraft:blast_furnace":29,"minecraft:lectern":30,"minecraft:bell":31,"minecraft:jigsaw":32,"minecraft:campfire":33,"minecraft:beehive":34,"minecraft:sculk_sensor":35,"minecraft:calibrated_sculk_sensor":36,"minecraft:sculk_catalyst":37,"minecraft:sculk_shrieker":38,"minecraft:chiseled_bookshelf":39,"minecraft:brushable_block":40,"minecraft:decorated_pot":41,"minecraft:crafter":42,"minecraft:trial_spawner":43,"minecraft:vault":44}
```

Comment:
Minified :vomiting_face:

Reply:
Im not fully convinced either, but if you want to discuss this maybe open a separate issue or do so on discord